azure-quantum/azure/quantum/target/honeywell.py

```diff
@@ -18,6 +18,35 @@ class Honeywell(Target):
         "honeywell.hqs-lt-s1-sim",
     )
 
+    # Get all one-qubit, two-qubit gates and measurement operations
+    GATES_1Q = [
```

Comment:
what about other 1 qubit gates not listed here?
it might be better to calculate if a gate is 1 or more qubits based on the number of targets.

Reply:
Sure, I'll have to check if the QASM ast contains info about the targets...

Reply:
I looked into this, it looks like [DAGCircuit](https://qiskit.org/documentation/stubs/qiskit.dagcircuit.DAGCircuit.html) has the method `DAGCircuit.op_nodes` that returns all the DAG nodes that are operations (`DAGOpNode`) which has a property `op` that includes the instruction name, number of qubits and number of classical bits. E.g.:
```python
print([n.op for n in dag.op_nodes()])
[Instruction(name='h', num_qubits=1, num_clbits=0, params=[]), Instruction(name='cx', num_qubits=2, num_clbits=0, params=[]), Instruction(name='cx', num_qubits=2, num_clbits=0, params=[]), Instruction(name='measure', num_qubits=1, num_clbits=1, params=[]), Instruction(name='measure', num_qubits=1, num_clbits=1, params=[]), Instruction(name='measure', num_qubits=1, num_clbits=1, params=[])]
```
I could find the number of gates of each type by filtering `dag.op_nodes()` by `num_qubits`, and pick out the `measure` and `reset` operations by name. Would something like that work?